components/script/dom/webidls/HTMLHyperlinkElementUtils.webidl

```diff
@@ -5,16 +5,17 @@
 
 // https://html.spec.whatwg.org/multipage/#htmlhyperlinkelementutils
 //[NoInterfaceObject/*, Exposed=Window*/]
```

Comment:
I think you need to uncomment this line partially so it becomes:

```
[NoInterfaceObject/*, Exposed=Window*/]
```

Because right now, it's looking in `components/script/dom/htmlhyperlinkelementutils.rs` for a concrete DOM structure, which shouldn't exist since this interface is marked as `NoInterfaceObject`